source/security/private-endpoints.txt

```diff
@@ -98,113 +76,109 @@ connection using either the App Services UI or the Admin API.
 
       .. procedure::
 
-         .. step:: Create an Interface Endpoint in Your VPC
-
-            To use a Private Endpoint, you must first create a VPC
-            interface endpoint in AWS. You can do this using the
-            :aws:`create-vpc-endpoint
-            </cli/latest/reference/ec2/create-vpc-endpoint>` command in the
-            AWS CLI. The UI will walk you through creating this command.
-
-            a. In the left navigation, click :guilabel:`App Settings`.
-
-            #. Click the :guilabel:`Private Endpoints` tab.
-
-            #. Click the :guilabel:`Add Private Endpoint` button.
-
-            #. Enter your **VPC ID** in the input box.
-
-            #. Enter your **Subnet IDs** as a comma-separated list in the input box.
+         .. step:: Create a VPC Interface Endpoint
 
-            Once you've entered this information, the fully constructed
-            AWS CLI command is displayed in the UI. Copy the command and
-            run it in your terminal. It should resemble the following:
-
-            .. code-block:: sh
-
-               aws ec2 create-vpc-endpoint \
-                 --vpc-endpoint-type Interface
-                 --service-name <App Services Endpoint Service Name> \
-                 --vpc-id <your-vpc-id> \
-                 --region <your-aws-deployment-region> \
-                 --subnet-ids <your-subnet-ids>
-
-            After you run the command, click :guilabel:`Next`.
+            a. In the App Services UI, navigate to :guilabel:`App Settings`,
+               then click the :guilabel:`Private Endpoints` tab.
+            #. Click :guilabel:`Add Private Endpoint`, then click
+               :guilabel:`General`.
+            #. Add your VPC ID and Subnet IDs (comma-separated list) in their
+               separate fields, then click :guilabel:`Next`.
+            #. Copy the generated :guilabel:`VPC Interface Endpoint Command`.
+               The command should look something like this:
+            
+               .. code-block:: sh
 
+                 aws ec2 create-vpc-endpoint \
+                   --vpc-endpoint-type Interface
+                   --service-name <App Services Endpoint Service Name> \
+                   --vpc-id <your-vpc-id> \
+                   --region <your-aws-deployment-region> \
+                   --subnet-ids <your-subnet-ids>
+            #. In your terminal, create a Sync VPC Endpoint with the generated
```

Comment:
I'm a little confused about the language we're using here, after reading the "Add a Sync Endpoint" section below. In ln 229, we say to create a Sync VPC Endpoint, and then we say to also create a General VPC Endpoint. So - should this one be a "General VPC Endpoint"? Or maybe a "VPC Endpoint"?

```suggestion
            #. In your terminal, create a General VPC Endpoint with the generated
```

Reply:
Yep. That should definitely be "General".